src/tableDrawer.ts

```diff
@@ -382,75 +382,120 @@ function printRow(
 
 function drawCellBorders(doc: DocHandler, cell: Cell, cursor: Pos) {
   const cellStyles = cell.styles
-
   doc.getDocument().setFillColor(doc.getDocument().getFillColor())
 
   if (typeof cellStyles.lineWidth === 'number') {
-    // prints normal cell border
+    // prints normal cell border using rect's stroke
     const fillStyle = getFillStyle(cellStyles.lineWidth, cellStyles.fillColor)
     if (fillStyle) {
       doc.rect(cell.x, cursor.y, cell.width, cell.height, fillStyle)
     }
   } else if (typeof cellStyles.lineWidth === 'object') {
-    doc.rect(cell.x, cursor.y, cell.width, cell.height, 'F')
-
-    const sides = Object.keys(cellStyles.lineWidth)
-    const lineWidth: any = cellStyles.lineWidth
-
-    sides.map((side: string) => {
-      const fillStyle = getFillStyle(lineWidth[side], cellStyles.fillColor)
-      drawBorderForSide(
-        doc,
-        cell,
-        cursor,
-        side,
-        fillStyle || 'S',
-        lineWidth[side]
-      )
-    })
+    drawCellBackground(doc, cell, cursor, cellStyles.fillColor)
+    drawBorders(doc, cell, cursor, cellStyles.fillColor, cellStyles.lineWidth)
   }
 }
 
-function drawBorderForSide(
+/**
+ * Prints cell background without borders and allows transparent color.
+ * @param doc
+ * @param cell
+ * @param cursor
+ * @param fillColor - `false` for transparent, `string` for color, other types will use "F" from jsPDF.rect
+ */
+function drawCellBackground(
   doc: DocHandler,
   cell: Cell,
   cursor: Pos,
-  side: string,
-  fillStyle: string,
-  lineWidth: number
+  fillColor: Color
+) {
+  const cellFillColor =
+    fillColor === false ? null : typeof fillColor !== 'string' ? 'F' : fillColor
```

Comment:
false should map to null
non strings to 'F'
strings can be passed to jsPdf.rect